client/components/AddProduct.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { postProduct } from '../store/product'
+
+class AddProduct extends Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            title: '',
+            description: '',
+            price: '',
+            quantity: '',
+            imageUrl: ''
+        }
+        this.handleChange = this.handleChange.bind(this)
+        this.handleSubmit = this.handleSubmit.bind(this)
+    }
+
+    handleChange(event){
+        let name = event.target.name
+        let value = event.target.value
+
+        this.setState({
+            [name]: value
+        })
+    }
+
+    handleSubmit(event){
+        event.preventDefault()
+        this.props.addProduct(this.state)
+    }
+
+    render(){
+        console.log(this.state)
+        return (
+            <div>
```

Comment:
I think we should add a Header like <h1>Add a new Product</h1>

Reply:
Done

Reply:
This is just to test functionality the form will in a modal later